modules/visual_script/visual_script_editor.cpp

```diff
@@ -3532,6 +3537,11 @@ VisualScriptEditor::VisualScriptEditor() {
 	left_vb->set_v_size_flags(SIZE_EXPAND_FILL);
 	//left_vb->set_custom_minimum_size(Size2(230, 1) * EDSCALE);
 
+	CheckButton *tool_script_check = memnew(CheckButton);
+	tool_script_check->set_text("Make Tool:");
```

Comment:
Should use `TTR()` to be translatable.
Also not sure how it looks like, but maybe the colon is redundant with it being a CheckButton? How about "Tool Script" as text?

Reply:
makes sense.

Reply:
This is still pending.

Reply:
oops, it's fixed now. :)